docs/en/rst/api/core/v1/bug.rst

```diff
@@ -1122,6 +1122,77 @@ This method can throw all the same errors as :ref:`rest_single_bug`, plus:
 * 604 (Summary Required)
   You did not specify a value for the "summary" argument.
 
+
+.. _rest_graph:
+
+Graph
+-----
+
+Return a graph of bug relationships.
+
+**Request**
+
+To return a simple json tree of the relationships for a given bug.
+
+.. code-block:: text
+
+  GET /rest/bug/1156/graph?type=json_tree
+
+To return a simple json tree that includes bug data in a manner similar to the "get" API,
+use bug_tree
+
+.. code-block:: text
+
+  GET /rest/bug/1156/graph?type=bug_tree
+
+The default relationship is the dependson to blocked relationship. To build the graph in the opposite direction:
+
+.. code-block:: text
+
+  GET /rest/bug/1156/graph?type=json_tree&relationship=dependencies:blocked,dependson
+
+**Response**
```

Comment:
Can you add a more detailed example of how this works? ie something like "if you have a bug 123 and you want to see the tree of blockers for the bug, and bug 123 is blocked by bug 234 and bug 345, this is how you make the API call and what it will look like". The JSON example only contains a tree with a single element so it's hard to visualize what this will look like in more complicated examples.

Reply:
I need to fix the docs anyway as the last refactor I did made the docs no longer match what is actually being returned. I will expand on the docs when I edit them as well. Thanks.